python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -52,8 +52,8 @@ def makeGangaListByRef(_list, preparable=False):
     result = GangaList()
     if len(_list) == 0:
         return result
-    temp_list = [stripProxy(element) for element in _list]
-    result._list = temp_list
+
+    result._list.extend([stripProxy(element) for element in _list])
```

Comment:
OK, after grepping the codebase only LHCbDataset abuses ```._list``` now so we can consider removing the intercept and warn users never to go near this internal parameter when modifying lists. (Which should be hidden behind a proxy anyway).